pkgs/development/compilers/gleam/default.nix

```diff
@@ -1,19 +1,19 @@
-{ stdenv, rustPlatform, fetchFromGitHub, Security }:
+{pkgs,  stdenv, rustPlatform, fetchFromGitHub, Security }:
```

Comment:
I don't see any usage of `pkgs` in this file. In nixpkgs, each derivation is called (in `pkgs/top-level/all-packages.nix`) as

```nix
{
  name = callPackage /path/to/derivation { };
}
```

where `callPackage` allows to select dependencies via derivation arguments. Thus, you can skip `pkgs`. If there is any reason for why it is added here, please let me know!

Reply:
Of, course, I just inserted it to have access to `pkgs.lib.fakeSha256` during updating the hashes, it was not meant to slip through…